frontend/src/pages/surveys/components/steps/goals-step/goals-step.tsx

```diff
@@ -0,0 +1,167 @@
+import { Button, Checkbox, Input } from '#libs/components/components.js';
+import { FIRST_INDEX } from '#libs/constants/index.constant.js';
```

Comment:
```suggestion
import { FIRST_INDEX } from '#libs/constants/constants.js';
```

Reply:
I think that index.constant is well, indicates that is a constant related to index